v0/sources/auth0/transform.js

```diff
@@ -38,43 +38,44 @@ const prepareGroupPayload = event => {
   return message;
 };
 
-function processEvent(event) {
-  // Dropping the event if type is not present
-  if (!event.type) {
-    return null;
-  }
-  const eventType = event.type;
-  // ss -> successful signup
-  if (eventType === "ss") {
-    return prepareIdentifyPayload(event);
-  }
-  // sapi -> Success API Operation
-  if (eventType === "sapi") {
-    if (
-      event.description &&
-      event.description === "Add members to an organization"
-    ) {
-      return prepareGroupPayload(event);
+function processEvent(eventList) {
```

Comment:
Perhaps this function name should be changed to `processEvents` now.

Reply:
I'd still recommend restricting it to just handle a single event though.

Reply:
function name updated to `processEvents`